ui/src/app/pages/data/cohort-review/annotation-definition-modals.component.tsx

```diff
@@ -252,14 +253,14 @@ export const EditAnnotationDefinitionsModal = withRouter(
           this.props;
         const {
           params: { ns, wsid, cid },
-        } = matchPath(location.pathname, {
+        } = matchPath<MatchParams>(location.pathname, {
           path: '/workspaces/:ns/:wsid/data/cohorts/:cid/reviews/:crid',
         });
         this.setState({ busy: true });
         await cohortAnnotationDefinitionApi().deleteCohortAnnotationDefinition(
           ns,
           wsid,
-          cid,
+          parseInt(cid, 10),
```

Comment:
MatchParams params are strings, but we need a `number` here for this Cohort ID

Reply:
@dolbeew could you check this file for correct conversions?

Reply:
I couldn't deploy locally to actually test it, getting a compile error (`Cannot find module 'terraui/out/Environments' or its corresponding type declarations.`), but looks like these changes should be fine.

Reply:
Thanks!  The `Environments` issue relates to #7319 so I suspect you'll have the same issue on `main`.  Peter has a tip in `#workbench-internal` which may help.  Please post there if you remain stuck.